astropy/utils/iers/iers.py

```diff
@@ -68,6 +69,18 @@
   conf.auto_max_age = None
 """
 
+
+def download_file(*args, **kwargs):
+    """
+    Overload astropy.utils.data.download_file within iers module to use a
+    custom (longer) wait time.  This just passes through ``*args`` and
+    ``**kwargs`` after temporarily setting the download_file remote timeout to
+    the local ``iers.conf.remote_timeout`` value.
+    """
+    with utils.data.conf.set_temp('remote_timeout', conf.remote_timeout):
+        return utils.data.download_file(*args, **kwargs)
```

Comment:
an alternative might be to do this instead of the ``with``:

```
kwargs['timeout'] = conf.remote_timeout()
return utils.data.download_file(*args, **kwargs)
```

That *should* be the same, but it might be a bit less prone to trouble than ``set_temp``?  (Although maybe this isn't worth worrying about)